BrowserCache_ConfigLabels.php

```diff
@@ -7,7 +7,11 @@ public function config_labels( $config_labels ) {
 				'browsercache.enabled' => __( 'Browser Cache:', 'w3-total-cache' ),
 				'browsercache.replace.exceptions' => __( 'Prevent caching exception list:', 'w3-total-cache' ),
 				'browsercache.no404wp' => __( 'Do not process 404 errors for static objects with WordPress', 'w3-total-cache' ),
-				'browsercache.no404wp.exceptions' => __( '404 error exception list:', 'w3-total-cache' ),
+				'browsercache.no404wp.response.cssjs' => __( '404 replacement response code for CSS/JS mime types', 'w3-total-cache' ),
+				'browsercache.no404wp.response.html' => __( '404 replacement response code for HTML mime types', 'w3-total-cache' ),
+				'browsercache.no404wp.response.other' => __( '404 replacement response code for Other mime types', 'w3-total-cache' ),
+				'browsercache.no404wp.response.exceptions' => __( '404 replacement response code for Exceptions URIs', 'w3-total-cache' ),
+				'browsercache.no404wp.exceptions' => __( '404 exception list:', 'w3-total-cache' ),
```

Comment:
You don't need to change old code if it's not relevant.

Reply:
I don't follow. The only old code here I changed was the exceptions label for better consistency with the new options and a few other wording changes to hopefully avoid convusion

Reply:
It's the change from "404 error exception list" to "404 exception list".